models.js

```diff
@@ -6,32 +6,88 @@ const dataGetTopics = () =>{
  })
 }
 
-const dataGetArticles = ()=>{
-   return db.query(`
-   SELECT articles.*,   
-   COUNT(comments.article_id) AS comment_count
-   FROM articles 
-   LEFT JOIN comments ON articles.article_id = comments.article_id
-   GROUP BY articles.article_id
-   ORDER BY comment_count DESC;
-   `)
-   .then((articles)=>{
-         return articles.rows
-      }
-   )
+//🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸
+//🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸
+//🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸🥸
+const dataGetArticles = (req,res) => {
+   const column = ["article_id","title","topic","author","body","created_at","votes","article_img_url"];
+   let querySortBy = req.query.sort_by
+   let queryOrderBy = req.query.order;
+
+   querySortBy = column.filter((e)=> e == querySortBy)
+
+   querySortBy.length == 0 ? querySortBy = "created_at" : querySortBy = querySortBy[0]
```

Comment:
I admire your use of conditional logic here, but this approach is going to make things hard to add other querys to. 
By passing the order by and sort by values directly from the controller, you would be able to set defaut values in the parameter declarations. 
like ` dataGetArticles = (orderBy = "ASC", sortBy = "created_at") => {...}`